sdks/python/apache_beam/typehints/trivial_inference.py

```diff
@@ -272,12 +279,12 @@ def infer_return_type_func(f, input_types, debug=False, depth=0):
     TypeInferenceError: if no type can be inferred.
   """
   if debug:
```

Comment:
I think we should remove debug code. Unrelated to this PR, but worth filing a JIRA.

Reply:
I've found the debug code to be pretty helpful while working on fixing the changes in tirivial inference. I know its a bit weird but if it doesn't slow things down for folks I assume the trivial inference stuff is going to break again in the future so having a debug flag we can turn on can be useful. That being said we could just replace it with a comment that there used to be debugging code back at [X] and then people can go back and see it if they need it but w/e.